reindexer/get_reindex_status.py

```diff
@@ -9,6 +9,7 @@
 from elasticsearch.exceptions import NotFoundError
 import humanize
 import tabulate
+from tenacity import retry, stop_after_attempt, wait_exponential
```

Comment:
Does this need to be added as a dependency somewhere?

Reply:
I don't think we have a requirements.txt for this.